src/codecoverage/bot/code_coverage_bot/taskcluster.py

```diff
@@ -44,15 +47,13 @@ def get_task(branch, revision, platform):
 
 
 def get_task_details(task_id):
-    r = requests.get(queue_base + 'task/{}'.format(task_id))
-    r.raise_for_status()
-    return r.json()
+    task_details_json = queue_service.task(task_id)
+    return task_details_json
```

Comment:
No need to have a temporary variable, you can just `return queue_service.task(task_id)`.